### PR TITLE
Feature/#79 update spinner

### PR DESCRIPTION
**Pull request by KolomiietsAn**
Replace gif image with plain CSS.

https://user-images.githubusercontent.com/93332463/142292783-ff2cb78b-d18b-4083-ba8a-e86e088e38b8.mov

### CODE REVIEW

**Review by Chubatik — APPROVED** [review]

File: src/components/Spinner/index.tsx [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import React, { Fragment } from 'react'
 
-import spinner from '../../assets/images/spinner.gif'
-
 import styles from './Spinner.module.scss'
 
 export const Spinner = () => (
   <Fragment>
```
Comment (vihopec) [inline src/components/Spinner/index.tsx:6]: we can remove fragment here (optional)

Comment (vihopec) [inline src/components/Spinner/index.tsx:6]: we can remove fragment here (optional)

**Review by vihopec — APPROVED** [review]

**Review by YanaGushlevskaya — APPROVED** [review]

**Review by irenlian — APPROVED** [review]

**Review by Chubatik — APPROVED** [review]
Please, resolve merge conflict

File: src/components/Spinner/index.tsx (hunk 1 above)
Comment (KolomiietsAn) [reply src/components/Spinner/index.tsx]: Thank you, removed

**Comment by KolomiietsAn** [comment]
> Please, resolve merge conflict

Thanks, fixed

**Pull request merged**